packages/sanity/src/core/create/components/CreateLinkedDocumentBannerContent.tsx

```diff
@@ -61,9 +61,7 @@ export function CreateLinkedDocumentBannerContent(props: CreateLinkedDocumentBan
         </Flex>
         <Stack space={4}>
           <Heading size={2}>{t('create-link-info-popover.header')}</Heading>
-          <Text size={1}>
-            <Translate t={t} i18nKey={'create-link-info-popover.text'} />
```

Comment:
No longer using the `Translate` component as the `create-link-info-popover.text` resource no longer contains markup